mysql-test/r/tc-heuristic-recover.result

```diff
@@ -20,5 +20,4 @@ Restarting with --tc-heuristic-recover=COMMIT
 SELECT * FROM t1;
 a
 1
-2
```

Comment:
The result file in PS-5.6 has this row -- meaning that server was able to recover the row after the server crash by using --tc-heuristic-recover=COMMIT during restart. So, it looks like there is some problem in recovery code in PXC. Until we find the cause, can we mark this as experimental or disable this test?

Reply:
You are right. There is something wrong here. 
It works fine on PS-5.6 with statement/row binlog format.
It always fails in the same way on PXC-5.6.
I've created PXC-3289.